addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1169,6 +1169,49 @@ QUnit.test('Post a message containing an email address followed by a mention on
     );
 });
 
+QUnit.test('Mention a partner with special character', async function (assert) {
```

Comment:
Which special character? Is it the `'` in the name? Please precise that in the name of test.